### PR TITLE
add a note about #8189 to the 0.77 release

### PR DESCRIPTION
**Pull request by amtoine**
i had no time to add this before the previous release, so here it is :yum: 

was originally added in https://github.com/nushell/nushell.github.io/pull/847

### CODE REVIEW

File: blog/2023-03-14-nushell_0_77.md [hunk 1]
```diff
@@ -190,6 +190,60 @@ Signatures:
...
+  │ # │ columnA │ columnB │
+  ├───┼─────────┼─────────┤
+  │ 0 │ A0*     │ B0      │
+  ╰───┴─────────┴─────────╯
+```
+
+> :bulb: **Note**  
+> the website has been modified accordingly with the same outputs.
+
 Make sure to browse the help messages of these commands. They contain fully functional examples thanks to pointing at www.example.com.
```
Comment (sholderbach) [inline blog/2023-03-14-nushell_0_77.md:247]: This should remain with the `http` commands

Comment (amtoine) [reply blog/2023-03-14-nushell_0_77.md]: very good catch, thanks :relieved: 

fixed back in b20cafb075232bb6e7b49bc6edee85674e25b620

**Review by sholderbach — APPROVED** [review]
Merci!

**Comment by amtoine** [comment]
thanks for guiding me with that one :relieved:

**Pull request merged**